pkg/test/harness.go

```diff
@@ -236,6 +236,11 @@ func (h *Harness) Config() (*rest.Config, error) {
 		return h.config, nil
 	}
 
+	if h.TestSuite.Config != nil {
```

Comment:
while this looks similar to the quick exit above...  it isn't the same IMO.   I would rather the `switch` below be the ways in which the `h.config` is configured... more importantly we should `h.T.Log` this.